### PR TITLE
add lbaas2, remove lbaas-connections

### PR DESCRIPTION
**Pull request by 404pilot**
Previously, I create lbaas uptime schema and lbaas connections schema.

Prod wants to add bandwidth usage and consolidate bandwdith, uptime and connections in a single event.

### CODE REVIEW

**Comment by usnavi** [comment]
You da man, @404pilot!  thanks!

File: tiamat/src/main/resources/nonStringAttrs.xsl [hunk 1]
```diff
@@ -143,8 +143,8 @@
       <schema key="http://docs.rackspace.com/usage/lbaas" version="1">
          <attributes>product/@avgConcurrentConnections,product/@avgConcurrentConnectionsSsl,product/@avgConcurrentConnectionsSum,product/@bandWidthIn,product/@bandWidthInSsl,product/@bandWidthOut,product/@bandWidthOutSsl,product/@hasSSLConnection,product/@numPolls,product/@numVips,product/@publicBandWidthInSum,product/@publicBandWidthOutSum</attributes>
       </schema>
-      <schema key="http://docs.rackspace.com/usage/lbaas/connections" version="2">
-         <attributes>product/@connections,product/@isSSL</attributes>
+      <schema key="http://docs.rackspace.com/usage/lbaas" version="2">
+         <attributes>product/@sslConnections product/@nonSslConnections product/@publicBandWidthOut</attributes>
```
Comment (shintasmith) [inline tiamat/src/main/resources/nonStringAttrs.xsl]: @404pilot this is the wrong format. The attributes should be comma separated, not space separated.

Did you copy the files from your ```standard-usage-schemas/target/xslt-artifacts``` to cloudfeeds-nabu? 

Can you please submit another PR to fix it? Thanks!
